core/qthelper.cpp

```diff
@@ -927,17 +927,17 @@ int parseGasMixHE(const QString &text)
 QString get_dive_duration_string(timestamp_t when, QString hourText, QString minutesText, QString secondsText, bool isFreeDive)
 {
 	int hrs, mins, fullmins, secs;
-	mins = (when + 59) / 60;
+	mins = (when + 30) / 60;
```

Comment:
I believe this was triggered by the fact that that's how many (some? some Suunto? I don't recall) dive computers did things... they always round up (and make your dive look longer) on the display.